core/controllers/profile.py

```diff
@@ -84,6 +97,12 @@ def get(self, username):
                 user_settings.first_contribution_msec
                 if user_settings.first_contribution_msec else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(
+                user_contributions.created_exploration_ids),
+            'edited_explorations_count': len(
+                user_contributions.edited_exploration_ids),
+            'created_exploration_summaries': created_exploration_summaries,
```

Comment:
Oh -- but why would you want to show the counts of deleted etc. explorations on the user's page, in any case? It would be weirder if the counts didn't match up with the exploration summary tiles that were displayed.